### DISCUSSION TITLE
Support arm architecture

### DISCUSSION THREAD

**Discussion by agungsetyawan** [coollabsio/coolify#261] [Feature Requests]
Will it support arm64? maybe will need it on a raspberry pi

**Comment by andrasbacsai** [comment]
Yes, I'm currently working on an ARM build.

**Reply by agungsetyawan** [reply]
Cool, thank you

**Comment by andrasbacsai** [comment]
Beta ARM support is already there. There are some pain points that will be solved in the next version. ATM, only 64bit is supported, but I plan to add older 32bit versions like armv7, as well.
